cli/import.full.yml

```diff
@@ -0,0 +1,138 @@
+graphar:                                              # Global configuration of imported data
```

Comment:
the import congfig files should move to testing repo too and it's better to use.the yaml as the only format to specify configurations.